### ISSUE TITLE
Update mac os devicelab prod pool fleet to new macos/xcode/ios versions

### ISSUE THREAD

**Issue by yusuf-goog** [flutter/flutter#94767]
Start upgrade of prod devicelab hosts to new macos (12.0.1), xcode and ios versions.

Once testing in prod pool is complete and all hosts have been upgraded, remove the multi-value dimensions from .ci.yaml for mac_ios.

**Comment by yusuf-goog** [comment]
due to issue with providing multi values in .ci.yaml, we are going to move mac devices to prod and do a switchover.

**Comment by godofredoc** [comment]
This is already done!
